packages/teams-js/src/public/calendar.ts

```diff
@@ -54,15 +65,24 @@ export namespace calendar {
     return ensureInitialized(runtime) && runtime.supports.calendar ? true : false;
   }
 
+  /** Open calendar item parameters. */
   export interface OpenCalendarItemParams {
+    /** A string representing the unique ID of the calendar item to be opened. */
     itemId: string;
   }
 
+  /** Compose meeting parameters */
+
   export interface ComposeMeetingParams {
+    /** An array of email addresses of the attendees to invite to the meeting. */
     attendees?: string[];
+    /** The start time of the meeting in ISO 8601 format. */
```

Comment:
>The start time of the meeting in ISO 8601 format. [](http://example.com/codeflow?start=8&length=49)

Should we look to use a more specific type here in the future than `string` which can enforce the ISO 8601 format?

Reply:
Currently we are using this `'MM/DD/YYYY HH:MM:SS'` format, and looking forward we should not be using string. The standard practice is to use standardized formats for representing dates and times, such as ISO 8601. This format provides a universal, unambiguous way of representing dates and times that is easily understood. It's often a good practice to store all date and time values in UTC, and to convert to and from local time zones as necessary for display purposes. 

At some point we should move away from current practice to standardized way though it's not required as we are encoding it into URI to execute deep link. I'll update the format in the comment for now.